force-app/main/default/customMetadata/Custom_Notification.Elevate_Advert.md-meta.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Elevate Advert</label>
+    <protected>false</protected>
```

Comment:
I believe this has to be protected = true if we want to be able to manage (and de-activate!) in the future!

Reply:
Good catch! Updated